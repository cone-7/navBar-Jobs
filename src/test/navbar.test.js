import React from 'react';
import NavBar from '../components/navbar.js';
import renderer from 'react-test-renderer';

test('render navbar', () => {
  const component = renderer.create(
    <NavBar searchText="test"></NavBar>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});
