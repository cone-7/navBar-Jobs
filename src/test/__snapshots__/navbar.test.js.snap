// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render navbar 1`] = `
<div
  className="nav-bar"
>
  <span>
    Start Bootstrap
  </span>
  <div
    className="icons"
  >
    <div />
    <div />
    <div />
  </div>
  <div
    className="search-div"
  >
    <button
      onClick={[Function]}
    />
    <input
      onChange={[Function]}
      placeholder="Search for..."
      type="text"
      value=""
    />
  </div>
</div>
`;
